addOns/reports/src/test/java/org/zaproxy/addon/reports/ExtensionReportsUnitTest.java

```diff
@@ -744,7 +744,7 @@ void shouldGenerateExpectedReport(String templateName) throws Exception {
         assertThat(cleanReport(report), is(equalTo(cleanReport(expected))));
     }
 
-    void checkAlert(JSONObject alert) {
+    void checkAlert(JSONObject alert, Boolean isJSONPlus) {
```

Comment:
JSONObject is outside our control but for our local variables/methods we should use proper camelCaps. So `isJsonPlus` here and at 793, etc.

Reply:
oh okay. I will change that.

Reply:
done